src/lib/core/typography/_typography.scss

```diff
@@ -42,17 +42,21 @@
 }
 
 // Adds the base typography styles, based on a config.
-@mixin mat-typography($config: mat-typography-config(), $selector: '.mat-typography') {
+// TODO(crisbeto): the headers need to handle margins as well.
+@mixin mat-base-typography($config, $selector: '.mat-typography') {
   .mat-h0, .mat-hero-header {
     @include mat-typography-level-to-styles($config, display-4);
+    letter-spacing: -0.05em;
```

Comment:
Add a comment somewhere explaining where the values came from, then? I know a year from now I'll be looking at this and be thinking "Where did these come from?!"